test/__snapshots__/test.js.snap

```diff
@@ -412,28 +418,44 @@ module.exports = {
         });
 
         this[impl][\\"attribute\\"] = V;
+      }
+    }
+    Object.defineProperties(Factory.prototype, {
+      method: {
+        configurable: true,
+        enumerable: true,
+        writable: true
       },
-
-      enumerable: true,
-      configurable: true
-    });
-
-    Object.defineProperty(Factory, \\"constant\\", {
-      value: 42,
-      enumerable: true
-    });
-    Object.defineProperty(Factory.prototype, \\"constant\\", {
-      value: 42,
-      enumerable: true
+      attribute: {
+        configurable: true,
+        enumerable: true
+      },
+      [Symbol.toStringTag]: {
+        value: \\"Factory\\",
+        configurable: true,
+        enumerable: false,
+        writable: false
+      },
+      constant: {
+        value: 42,
+        configurable: false,
+        enumerable: true,
+        writable: false
+      }
     });
-
-    Object.defineProperty(Factory.prototype, Symbol.toStringTag, {
-      value: \\"Factory\\",
-      writable: false,
-      enumerable: false,
-      configurable: true
+    Object.defineProperties(Factory, {
+      prototype: {
```

Comment:
Class declarations by default generate a prototype with this descriptor, so this is not necessary.

Reply:
Interesting, I was not aware of that.